tests/integration/test_integration_watcher.py

```diff
@@ -1,20 +1,32 @@
 from fastapi import FastAPI
 from starlette.testclient import TestClient
 
+from antarest.core.tasks.model import TaskStatus
 
-def test_integration_xpansion(app: FastAPI, tmp_path: str):
-    client = TestClient(app, raise_server_exceptions=False)
+
+def test_integration_watcher(app: FastAPI, tmp_path: str):
+    client = TestClient(app)
     res = client.post(
```

Comment:
You can consider using the `admin_access_token` fixture to retrive admin credentials.